todo/src/components/todolist.js

```diff
@@ -1,42 +1,76 @@
-import React, {Component } from 'react';
+import React, {Component} from 'react';
 import { connect } from 'react-redux';
-import { addTodo } from '../actions'
-
+import {addTodo, toggleTodo} from '../actions'
 
 class TodoList extends Component {
     constructor() {
         super();
         this.state = {
-            text: ''
+            todo: ''
         }
-    }
+    };
 
-    inputHandler = (e) => {
-        this.setState({[e.target.name]: e.target.value})
-        console.log(this.state.text);
+    handleTodoChange = e => {
+        this.setState({ todo: e.target.value })
     }
 
-    addToDoHandler = (e) => {
+    addTodoItem = (e) => {
         e.preventDefault();
-        const newTodo = {text: this.state.text, completed: false, id: Date.now()}
-        this.props.addTodo(newTodo);
-        this.setState({text: ''});
+        this.props.addTodo(this.state.todo);
+        this.setState({todo: ''})
+    }
+
+    toggleCompleted = () => {
+
     }
 
     render() {
-        console.log('Todo List', this.props)
-        return (
-            <form>
-                <input type="text" value={this.state.text} name='text' onChange={this.inputHandler}/>
-                <button onClick={this.addToDoHandler}>Add</button>
-            </form>
-            // {this.props}
+        console.log(this.props)
```

Comment:
Try to remove `console.log()`s when no longer needed. Totally ok if still debugging 😄